packages/components/psammead-episode-list/README.md

```diff
@@ -62,11 +62,11 @@ The `EpisodeList.Title` component is responsible for styling text to be presente
 
 The `EpisodeList.Description` component is responsible for styling text to be presented the description of an individual episode. This component is intended to be a child of `EpisodeList.Episode`
 
-### Components: `EpisodeList.Metadata`
+### Components: `EpisodeList.DateTimeDuration`
 
-The `EpisodeList.Metadata` component is responsible for styling text to be presented as supplemental metadata for an individual episode. This component is intended to be a child of `EpisodeList.Episode`.
+The `EpisodeList.DateTimeDuration` component is responsible for styling text to be presented as supplemental dateTime or duration metadata for an individual episode. This component is intended to be a child of `EpisodeList.Episode`.
```

Comment:
We should also update the jsx in Basic usage and Advanced Usage. L85 and L17 of this file

Reply:
Great spot 👍